common.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/usr/bin/env bash -e
```

Comment:
This doesn't pass shellcheck. You can't have more than one argument in a shebang. If you use env you need to set -e elsewhere in the file.

Reply:
Missed that one